src/foam/u2/view/PasswordView.js

```diff
@@ -79,7 +79,7 @@ foam.CLASS({
 
       this.addClass(this.myClass()).start().
         addClass('input-field-container').
-        start(foam.u2.tag.Input, {
+        start(foam.u2.TextField, {
```

Comment:
Ideally we should be doing `requires: [ 'foam.u2.TextField' ]` and then `start(this.TextField` but that's a minor issue, something to keep in mind.

Reply:
Resolved by ae634b2.